guides/unmanaged_devices.md

```diff
@@ -0,0 +1,276 @@
+# Using Unmanaged Devices
+
+A guide for everyone that needs to use unmanaged devices.
+
+### What is this about?
+This guide is about the appropriate steps you should take when using devices that are not managed by the normal
+organisational IT controls.
+
+Primarily that means developer's laptops.
```

Comment:
Lots of other people use unmanaged laptops - for example, researchers, designers, testers. I suggest we either make this more general or specify more examples of roles, so that people see it applies to them.

```suggestion
That means "off-network" laptops used, for example, by developers, testers and designers on digital services.
```

Reply:
Changed, good spot 👍